neon_core/__init__.py

```diff
@@ -82,6 +90,17 @@ def setup_ovos_config():
 
 setup_ovos_config()
 
+# make ovos-core Configuration.get() load neon.conf
+# TODO ovos-core does not yet support yaml configs, once it does
+#  Configuration.get() will be made to load the existing neon config files,
+#  for now it simply provides correct default values
+setup_ovos_core_config()
+
+neon_config_path = join(xdg.BaseDirectory.save_config_path("neon"),
+                        "neon.conf")
+write_mycroft_compatible_config(neon_config_path)
+LOG.info(f"{neon_config_path} will be overwritten with Neon YAML config contents.")
```

Comment:
this will be logged every time right? it would be better if it only logged when the file was actually replaced, so maybe behind a check if it exists? 

otoh we have no way of knowing if that file is user defined or already contains the neon config... so after the first overwrite this log will not be accurate either way

Don't block the PR on this, leave it for now, but maybe add a TODO ?